injector/network_disruption.go

```diff
@@ -30,7 +30,7 @@ import (
 type linkOperation func([]string, string, uint32) error
 
 // tcPriority the lowest priority set by tc automatically when adding a tc filter
-var tcPriority = uint32(49149)
+var tcPriority = uint32(1000)
```

Comment:
Do we know where the old value comes from? Just to be sure it has no side effect to start from 1000 now.

Reply:
the old value comes from the automatically set priority. I took it because of this, but it's totally safe to use this instead